config.js

```diff
@@ -1,10 +1,18 @@
 module.exports = {
     GUILD_IDS: {
-        "798975165342023700": {
-            repo: "sourcecred/dev-cred", 
-            branch: "payout-bot-test", 
+        "956551108061446244": {
+            repo: "nation3/nationcred-instance", 
+            branch: "activation-bot-test", 
+            discordBaseURL: "https://forum.nation3.org",
```

Comment:
@johnmark13 Why is the `discordBaseURL` a link to the forum?

Reply:
Actually, now we removed Discourse we do not need that at all, so I will remove, good spot.